### PR TITLE
Add two triggers

### PR DESCRIPTION
**Pull request by ItsPugle**
Let's talk with management?
Who wants to talk with upstream?

### CODE REVIEW

File: src/data/triggers.json [hunk 1]
```diff
@@ -131,4 +131,6 @@
   "upmarket",
   "value (add|proposition)",
   "verb(iage|age)"
+  "with upstream"
```
Comment (okize) [inline src/data/triggers.json]: hi @ItsPugle, thanks for the PR!

I think you'll need some trailing commas on lines `133` & `134` or the json won't parse

Comment (ItsPugle) [reply src/data/triggers.json]: Changed - How does that look now? I'm not familiar with json so yeah....

**Comment by okize** [comment]
thanks @ItsPugle! published as `v1.5.10`

**Comment by ItsPugle** [comment]
Sweet! Thank you <3

**Pull request merged**